src/hera_display_games/use_controller/__init__.py

```diff
@@ -5,13 +5,35 @@
 import numpy as np
 import asyncio
 from hera_display_games.core import mechanics, keymapper
-import time
 import click
 
 
-async def recolor(sprite):
-    time.sleep(10)
-    return np.random.randint(0, 255, size=3).tolist()
+async def recolor(sprite, board):
+    while True:
+        await asyncio.sleep(5)
+        sprite.color = np.random.randint(0, 255, size=3).tolist()
+        board.draw()
+
+
+async def move_sprite(device, sprite, board):
+    while True:
+        response = await keymapper.map_movement(device)
+        if response in ["ul", "ur", "dl", "dr", "r", "l"]:
+            sprite.move(response)
+            board.draw()
+        elif response in ["r-trigger", "l-trigger"]:
+            sprite.color = np.random.randint(0, 255, size=3).astype(int).tolist()
+            board.draw()
```

Comment:
Do the `board.draw()` calls need to be in the input loops now? We've been moving towards a concept where the board resolves mechanics and draws when it's ready, so ideally I'd like to have one `board.draw()` call done in a master loop.

Reply:
ah this. I did this mainly to ensure that all the parts were working even with different cadences. I do not see a reason it couldn't work the way you envisioned though. As long as the `board.draw()` cadence is at least as fast as any expected input/updates/etc. I'll try to change this around.

Reply:
I moved all the board drawing into its own task. Sounds like what you wanted. This is basically just an example or proof of concept for _how_ to get things to work in the background.

Reply:
Cool, I think this works. Steven and I will work on porting it into his branch once this one is merged in.